### PR TITLE
Fixed #33163 -- Added example of connection signal handlers in AppConfig.ready() to docs.

### PR DESCRIPTION
**Pull request by roxanebellot**
See [ticket](https://code.djangoproject.com/ticket/33163#ticket) and [forum post](https://forum.djangoproject.com/t/load-signals-automatically/9666)

I tried to keep as much as existing naming and  formulation as I could.

When it is ok with you, I can also do a French translation

### CODE REVIEW

File: docs/topics/signals.txt [hunk 1]
```diff
@@ -181,6 +162,44 @@ Different signals use different objects as their senders; you'll need to consult
 the :doc:`built-in signal documentation </ref/signals>` for details of each
 particular signal.
 
+
```
Comment (felixxm) [inline docs/topics/signals.txt:165]: Chop blank line.

File: docs/topics/signals.txt [hunk 2]
```diff
@@ -181,6 +162,44 @@ Different signals use different objects as their senders; you'll need to consult
...
+        ...
+
+        def ready(self):
+            # implicitly connect @receiver signal handlers
+            from . import signals  
+            # explicitly connect custom signal handlers
+            signals.request_finished.connect(signals.my_callback)
+
+
+
```
Comment (felixxm) [inline docs/topics/signals.txt:196]: Chop blank lines.

File: docs/topics/signals.txt [hunk 3]
```diff
@@ -181,6 +162,44 @@ Different signals use different objects as their senders; you'll need to consult
...
+The :meth:`~django.apps.AppConfig.ready` method may be executed more than
+once during testing, so you may want to :ref:`guard your signals from
+duplication <preventing-duplicate-signals>`, especially if you're planning
+to send them within tests.
+
+
```
Comment (felixxm) [inline docs/topics/signals.txt:202]: Chop blank line.

**Review by felixxm — COMMENTED** [review]
@roxanebellot Thanks for this patch :+1: I don't think that a separate section is needed. I would add an example to the existing admonition. Please try to avoid unnecessary blank lines.

**Comment by roxanebellot** [comment]
Hi !

Thank ou for your interest, blank lines are chopped :heavy_check_mark: 

Can you confirm that a subsection is unfit compared to using an admonition ? We would loose the benefit of the anchor and maybe legibility too (it would be a big green zone)

Also, i am having doubts about the title "Where should this code live" but if it's ok with you, it's ok with me.

**Review by blockchainGuru1018 — APPROVED** [review]

**Comment by felixxm** [comment]
@roxanebellot Thanks for updates :+1: Welcome aboard :boat: 

I pushed small edits.

**Comment by adamchainz** [comment]
Thanks @roxanebellot and @felixxm !

**Pull request merged**